docs/source/guides/containerization.rst

```diff
@@ -0,0 +1,602 @@
+================
+Containerization
```

Comment:
rename this title to "Custom Dockerfile"?

This guide is not specific to Containerization - it's about advanced customization of the generated dockerfile

Reply:
sure. I don't have strong opinion on this.